src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo: { completed, title, user } }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': completed === true,
+  })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    <UserInfo user={user} />
```

Comment:
```suggestion
    {user && <UserInfo user={user} />}
```
possible case when user will be null.
better to render UserInfo only when user exists.

Reply:
thank you for putting my attention to this! it definitely makes sense.